streaming/text/enwiki.py

```diff
@@ -14,24 +14,24 @@ class EnWiki(Dataset):
     """Implementation of the English Wikipedia 2020-01-01 streaming dataset.
 
     Args:
-        local (str): Local dataset directory where shards are cached by split.
-        remote (str, optional): Download shards from this remote path or directory. If None, this
-            rank and worker's partition of the dataset must all exist locally. Defaults to
-            ``None``.
-        split (str, optional): Which dataset split to use, if any. Defaults to ``None``.
-        shuffle (bool): Whether to shuffle the samples while iterating. Defaults to ``True``.
-        prefetch (int, optional): Target number of samples remaining to prefetch while iterating.
+        local (str): Local filesystem directory where dataset is cached during operation.
+        remote (str, optional): Remote directory (S3 or local filesystem) where dataset is stored.
             Defaults to ``None``.
-        keep_zip (bool, optional): Whether to keep or delete the compressed file when
-            decompressing downloaded shards. If set to None, keep iff remote is local. Defaults to
+        split (str, optional): The dataset split to use, either 'train' or 'val'. Defaults to
             ``None``.
+        shuffle (bool): Whether to shuffle the samples in this dataset. Defaults to
```

Comment:
same commentary as above

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/42/commits/c023c30b7f55b89dd9f6526480c0c5a272940bba.